src/react/components/pages/modelCompose/modelCompose.tsx

```diff
@@ -0,0 +1,647 @@
+            constants.apiPreviewPath,
+        ) : url.resolve(
+            this.props.project.apiUriBase,
+            nextLink,
+        );
+        const config = {
+            headers: {
+                "Access-Control-Allow-Origin": "*",
+                "withCredentials": "true",
+            },
+        };
+
+        try {
+            return await ServiceHelper.getWithAutoRetry(
+                baseURL,
+                config,
+                this.props.project.apiKey as string,
+            );
+        } catch (err) {
+            this.setState({
+                isLoading: false,
+            });
+            console.log(err);
+            ServiceHelper.handleServiceError(err);
+        }
+    }
+
+    private handleColumnClick = (event: React.MouseEvent<HTMLElement>, column: IColumn): void => {
+        const {columns, modelList} = this.state;
+        const newColumns: IColumn[] = columns.slice();
+        const currColumn: IColumn = newColumns.filter((col) => column.key === col.key)[0];
+        newColumns.forEach((newCol: IColumn) => {
+            if (newCol === currColumn) {
+                currColumn.isSortedDescending = !currColumn.isSortedDescending;
+                currColumn.isSorted = true;
+            } else {
+                newCol.isSorted = false;
+                newCol.isSortedDescending = true;
+            }
+        });
+        const newList = this.copyAndSort(modelList, currColumn.fieldName!, currColumn.isSortedDescending);
+        this.allModels = newList;
+        this.setState({
+            columns: newColumns,
+            modelList: newList,
+        });
+    }
+
+    private copyAndSort(modelList: IModel[], columnKey: string, isSortedDescending?: boolean): IModel[] {
+        const key = columnKey;
+        if (key === "createdatetime" || key === "lastupdateddatetime") {
+            return (modelList.slice(0)
+            .sort((a, b): number => {
+                if (isSortedDescending) {
+                    if ((new Date(a.createdDateTime)).getTime() < (new Date(b.createdDateTime)).getTime()) {
+                        return 1;
+                    } else {
+                        return -1;
+                    }
+                } else {
+                    if ((new Date(a.createdDateTime)).getTime() > (new Date(b.createdDateTime)).getTime()) {
+                        return 1;
+                    } else {
+                        return -1;
+                    }
+                }
+            }));
+        } else {
+            return (modelList.slice(0)
+            .sort((a: IModel, b: IModel) => ((isSortedDescending ? a[key] < b[key] : a[key] > b[key]) ? 1 : -1 )));
+        }
+    }
+
+    private handleSelection = (): string => {
+        return "item selected";
+    }
+
+    /** Handle filter when text changes in filter field */
+    private onTextChange = (ev: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>, text: string): void => {
```

Comment:
I think filter should not be case sensitive, because for query miami we're not getting results, but we do for Miami